docs/development/workflow/virtual_pythons.rst

```diff
@@ -129,6 +129,11 @@ environment.
       `documentation for the conda command`_ and the
       `blog post announcing anaconda environments`_.
 
+    + If astropy is installed in your environment, you may need to uninstall it
+      in order for the development version to install properly. You can do this       with the following command::
+
+        conda uninstall ENV astropy
```

Comment:
I think this should just be ``conda uninstall astropy``

Reply:
Things also get tricky if you did a `conda install astropy` *and* `pip install astropy` but that's probably out of scope here. :grimacing: